package.json

```diff
@@ -22,11 +22,11 @@
   "homepage": "https://github.com/praekelt/go-ndoh",
   "dependencies": {
     "vumigo_v02": "~0.2",
-    "q": "~0.9",
+    "q": "~0.9.7",
     "lodash": "~2.4.1",
-    "moment": "~2.6.0",
-    "go-jsbox-metrics-helper": "~0.1.1",
-    "libxmljs": "0.8.x"
+    "moment": "~2.6",
+    "go-jsbox-metrics-helper": "^0.1.0",
+    "libxmljs": "~0.8.1"
```

Comment:
Ah, these were just copied across from Vumi Go's package.json to be safe. I think that's okay.